src/Card/index.css

```diff
@@ -6,16 +6,31 @@
     margin: 60px 30px;
     color: green;
     display: inline-block;
+    overflow: auto;
   }
   .card p:first-child {
+    height: auto;
     padding-bottom: 10px;
     border-bottom: 1px solid red;
-  }
-  input {
-    float: right;
+    display: flex;
+    justify-content: space-between;
   }
   .card-checked {
     border: none;
     border-radius: 5px;
     background-color: pink;
   }
+  .non-active {
+    display: none;
+  }
+  .active {
+    display: block;
+  }
```

Comment:
This is a bad practice to render 'dead' elements. (rendered but hidden). Some hacker could reach out to them via dev tools and raise some unpredicted behavior, in worst case - make app crashed. 
If you do not need element rendered - simply do not render it. Use if or ternary operators to return only required nodes.